src/main/java/org/broadinstitute/hellbender/tools/walkers/mutect/Mutect2Engine.java

```diff
@@ -578,28 +407,22 @@ protected int getCountOfNonRefEvents(final ReadPileup pileup, final byte refBase
     }
 
     private double calculateF(final ReadPileup pileup, final byte refBase, final byte minBaseQual) {
```

Comment:
I missed filtering by base quality for `total count`. I agree with you, for loop is better.